modules/storages/spec/common/storages/peripherals/storage_interaction/nextcloud/copy_template_folder_command_spec.rb

```diff
@@ -115,8 +115,7 @@
         result = subject.call(source_path:, destination_path:)
 
         expect(result).to be_success
-        expect(result.result[:id]).to eq('349')
-        expect(result.result[:url]).to eq(destination_url)
```

Comment:
🟢  Q: anything changed with `destination_url`? i.e. why are we not asserting the expected result anymore?

Reply:
Oh, it changed place and therefore is now tested elsewhere.